### PR TITLE
Fixed readme formatting

### PR DESCRIPTION
**Pull request by ro0NL**
@keradus WDYT?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -175,657 +175,438 @@ automatically fix anything:
...
-                         between cast and variable.
-
-* **class_definition** [@PSR2, @Symfony]
-                         Whitespace around the key
-                         words of a class, trait or
-                         interfaces definition should
-                         be one space.
+   Align equals symbols in consecutive lines.
+
+* **binary_operator_spaces [@Symfony]**
```
Comment (keradus) [inline README.rst]:
```diff
-* **binary_operator_spaces** [@Symfony]
+* **binary_operator_spaces [@Symfony]**
```

why is that change?

**Comment by keradus** [comment]
github becomes crappy lately...

I retargetted your PR to 1.12.
Please rebase to not have all master commits ;)

I like the idea in general, let somebody do the review after rebase ;)

**Comment by ro0NL** [comment]
Not sure you can merge this upstream easily.. i tried to rebase 1.12 but besides the file rename, the logic is more or less different, ie. conflicting.

How about a separate PR?

File: README.rst (hunk 1 above)
Comment (ro0NL) [inline README.rst]: Revised.

**Comment by keradus** [comment]
used while merging 1.x->2.x
Thanks @ro0NL

**Pull request closed without merging**